### ISSUE TITLE
Missing voiceover translation for buttons

### ISSUE THREAD

**Issue by chrisbonn86** [wireapp/wire-ios#5037]
## The problem

Unfortunately, some buttons in message are of the Wire app are not usable for blind people. The voiceover label of the buttons is "Taste" in german, i guess it's "button" or "key" in english.
Actually there is an voiceover translation for the camera icon, the microfone icon and the location icon. But it is missing for the "@" icon, the paint area, the ping, the video recorder, the message timeout and the special menu for formatting messages aswell as the formatting icon.
Would be great if it could be added to the app.

## Environment

* Wire version that exhibits the issue:
3.77
